src/components/Footer/index.tsx

```diff
@@ -3,7 +3,7 @@ import { Box, Typography } from '@mui/material';
 import Line from '@shared/Line';
 import { getNavLinks } from '@components/Header/navLinks';
 
-import MicrosoftIcon from '@assets/icons/microsoft.svg?react';
+import MicrosoftIcon from '@assets/icons/microsoft-icon.svg?react';
 import GithubIcon from '@assets/icons/github.svg?react';
```

Comment:
Change the names of any SVG icons that don't have the suffix '-icon', such as Github.svg etc.